lib/rubocop/cli/command/auto_genenerate_config.rb

```diff
@@ -4,6 +4,9 @@ module RuboCop
   class CLI
     module Command
       # Generate a configuration file acting as a TODO list.
+      # @api private
```

Comment:
I guess the duplicate is an oversight, right?

Reply:
Oops, search and replace error...